flixel/addons/weapon/FlxWeapon.hx

```diff
@@ -86,22 +86,6 @@ class FlxTypedWeapon<TBullet:FlxBullet>
 	 */
 	public var parent(default, null):FlxSprite;
 
-	/**
-	 * Whether to fire bullets in the direction the `parent` is facing (e.g. `UP`, `DOWN`, `LEFT`, `RIGHT`). Used only when `fireFrom == PARENT`.
-	 */
-	public var useParentDirection:Bool;
-
-	/**
-	 * If `parent == null`, the Weapon will fire from a fixed position on the screen, like in the game Missile Command.
-	 */
-	public var firePosition(default, null):FlxBounds<FlxPoint>;
-
-	/**
-	 * A value to use to offset a bullet's position when it is fired.
-	 * Can be used to, for example, line a bullet up with the "nose" of a space ship.
-	 */
-	public var positionOffset(default, null):FlxPoint;
```

Comment:
same here, just deprecate them

Reply:
I'm instead making them actually access the fields in the enums, but there is an issue: `positionOffset` is of type `FlxPoint` whereas `FlxWeaponFireFrom.POSITION.position` is of type `FlxBounds<FlxPoint>`. What should I do about this?
Also, do you know whether it's possible to change the fields of an enum without making another "instance" of one?